src/js/components/stock-movement-wizard/StockMovement.jsx

```diff
@@ -113,26 +116,31 @@ class StockMovements extends Component {
           previousPage={this.previousPage}
           goToPage={this.goToPage}
           onSubmit={this.nextPage}
+          showOnly={showOnly}
```

Comment:
I do not understand one thing here. Previously on each page component showOnly was equal to `queryString.parse(window.location.search).type === 'REQUEST'; `
and now its:
`this.state.values.origin && this.props.currentLocation.id !== this.state.values.origin.id && !this.props.isUserAdmin;`
why you are no longer checking for "request" type?
(Luckily this type of issues will be resolved when SM workflows refactoring will be finished)

Reply:
I still check it in StockMovement and AddItemsPage components. We only need to check it when user submits request. Once it's submitted it cannot be edited from location other than origin. Previously I've check the url because I've thought that we're going to add this "type=REQUEST" parameter to url each time users want to check the status of submitted request but we don't add it. 
Yes, we should get rid of such issues then.

Reply:
Ok, then it LGTM